problems/2.functions.js

```diff
@@ -20,17 +21,21 @@ function greeting() {
 * @returns {null}
 * @modifies {global.globalCount}
 */
-function incrementGlobalCount() {
+function incrementGlobalCount(val) {
+  let val=0
```

Comment:
To update a variable you need to do reassignment and thus need the assignment operator (=). 
global.globalCount += val